src/styles/utils/_reset.scss

```
@@ -0,0 +1,8 @@
+* {
+  margin: 0;
+  padding: 0;
+}
```

do not set margin and padding to all elements. add the style only fox needed
```suggestion
* {
}
```